### ISSUE TITLE
Tabs don't give focus to content when opening panel.

### ISSUE THREAD

**Issue by CGTS** [zurb/foundation-sites#9767]
#### How to reproduce this bug:

1. Use Tabs from [http://foundation.zurb.com/sites/docs/tabs.html](url) . Make sure you add a link into the content panel.
2. Use keyboard to open Tab and and try access a link within the panel with your keyboard.

#### What should happen:

When I use a keyboard to open a Tab, The next <kbd>tab</kbd> should take me to a focusable element within the content

#### What happened instead:

After selecting the Tab I am not able to navigate into the content with a keyboard. Instead of moving focus to the Tab content I move move to the next Tab item.

#### Test case:

https://codepen.io/CGTS/pen/XpyZYK

**Comment by IamManchanda** [comment]
You are right ... will be a great addition considering accessibility

**Comment by CGTS** [comment]
Anyone have any ideas how to implement something like this in the interim?

**Comment by IamManchanda** [comment]
poke @kball => http://jsfiddle.net/Z7P5L/4/

**Comment by CGTS** [comment]
Thanks I'll take a look at this.

**Comment by IamManchanda** [comment]
Special Note: This functionality should be added in both ..... tabs and accordions ( and of course responsive accordion tabs also )

**Comment by kball** [comment]
@CGTS are you taking a look at this? Otherwise maybe @Owlbertz can take a look, as he's our keyboard expert

**Comment by CGTS** [comment]
@kball Unfortunately, I wouldn't be much use with javascript. I'd be happy to help test it.

**Comment by Owlbertz** [comment]
Hi,
it seems like the there is a something odd with preventing the default action using space. I will have a look at this.

**Comment by kball** [comment]
@Owlbertz I'm inclined to go with your opinion... Do you want to implement?

**Comment by CGTS** [comment]
I just spoke with a guy at here at work who is a [VA Trusted Tester](https://www.dhs.gov/trusted-tester) (pending) and he says either solution will pass the VA's accessibility testing (it tends to be more strict). He prefers the first solution though (arrow nav on tabs).

**Comment by kball** [comment]
@Owlbertz can you take a pass on this?

**Comment by Owlbertz** [comment]
Sorry for the late response, I made a PR addressing this issue. Please have a look and check if the behavior works as intended, @CGTS.

**Comment by kball** [comment]
Fixed via #9960 going with the first solution (arrow nav on tabs)

**Comment by CGTS** [comment]
@kball @Owlbertz Can we reopen this issue? We recently sent a course through the 508 accessibility office of the Dept. of Veterans Affairs for certification. Unfortunately, the tabs component did not pass. Below is feedback we received. 

> When the page tabs are activated, there is no way to move to the tab panel content with the keyboard. The arrow keys move the user to the next accordion tab while the tab key itself has nothing on which to focus within the tab panels. This will make it impossible for non-sighted users to learn what is on the  tab panels.

> Manage the focus so that when an accordion tab is activated the focus immediately moves to the tab panel content.

What would you suggest as the best way to add the ability to tab into the content after the tab has been given focus? Similar to [this example](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/tabs/tabs-1/tabs.html).

**Comment by ncoden** [comment]
@CGTS See https://github.com/zurb/foundation-sites/issues/11225

This issue will stay closed as the requested changed in it has been merged.

**Comment by CGTS** [comment]
Understood. I appreciate the creation of the new issue.
